packages/reason-css-parser/lib/Parser.re

```diff
@@ -146,6 +146,19 @@ and color_stop_length = [%value.rec
 and color_stop_list = [%value.rec
   "[<linear-color-stop> ]? [ ',' <linear-color-stop> ]# ',' <linear-color-stop>"
 ]
+and hue_interpolation_method = [%value.rec
+  "[ 'shorter' | 'longer' | 'increasing' | 'decreasing' ] 'hue' "
+]
+and polar_color_space = [%value.rec " 'hsl' | 'hwb' | 'lch' | 'oklch' "]
+and rectangular_color_space = [%value.rec
+  " 'srgb' | 'srgb-linear' | 'display-p3' | 'a98-rgb' | 'prophoto-rgb' | 'rec2020' | 'lab' | 'oklab' | 'xyz' | 'xyz-d50' | 'xyz-d65' "
+]
+and color_interpolation_method = [%value.rec
+  " 'in' | [ <rectangular-color-space> | <polar-color-space> <hue-interpolation-method>? ] "
+]
+and function_color_mix = [%value.rec
+  "color-mix(<color-interpolation-method> ',' [ [ <color> && <percentage [0,100]>? ] ]#{2})"
```

Comment:
Where `percentage [0,100]` comes from?

Reply:
It's part of the [syntax](https://drafts.csswg.org/css-color-5/#color-mix).

Reply:
but It's not supported by our parser AFAIK